todo_app/app.py

```diff
@@ -83,11 +68,27 @@ def get_sort_parameters():
 @app.route('/', methods = ['GET'])
 def index():
     
+    member = get_member('alanjknight@hotmail.com')
+    hydrate_member(member)
+
     sort_col, last_sort_col, sort_reverse, sort_dir = get_sort_parameters()
+
+    session['sort_col']=sort_col
+    session['last_sort_col'] = last_sort_col
+    session['sort_reverse']=sort_reverse
+    session['sort_dir']=sort_dir
     
-    items = get_items()
-    items = sorted(items,key=itemgetter(sort_col), reverse=sort_reverse)
+    
+    if(sort_col=='id'):
+        member.board_list[0].item_list = sorted(member.board_list[0].item_list,key=lambda item: item.id, reverse=sort_reverse)
+    elif(sort_col=='title'):
+        member.board_list[0].item_list = sorted(member.board_list[0].item_list,key=lambda item: item.title, reverse=sort_reverse)
+    elif(sort_col=='status'):
+        member.board_list[0].item_list = sorted(member.board_list[0].item_list,key=lambda item: item.status, reverse=sort_reverse)
+    elif(sort_col=='target_date'):
+        member.board_list[0].item_list = sorted(member.board_list[0].item_list,key=lambda item: item.target_date, reverse=sort_reverse)        
```

Comment:
We could make this a little more readable by splitting out a lot of the repeated parts;
```python
sorting_method = None
    if(sort_col=='id'):
        sorting_method = key=lambda item: item.id
    elif(sort_col=='title'):
        sorting_method = key=lambda item: item.title
    elif(sort_col=='status'):
        sorting_method = key=lambda item: item.status
    elif(sort_col=='target_date'):
        sorting_method = key=lambda item: item.target_date
    if sorting_method:
        member.board_list[0].item_list = sorted(member.board_list[0].item_list,key=sorting_method, reverse=sort_reverse)
```

It also feels ripe for potentially pulling out into a function, at least to pick which sorting method to use, something like:

```python
    sorting_method = get_sorting_method()
    if sorting_method:
        member.board_list[0].item_list = sorted(member.board_list[0].item_list,key=sorting_method, reverse=sort_reverse)
...


def get_sorting_method():
    <appropriate code here>
```